eclipse/src/saros/ui/actions/OpenChatAction.java

```diff
@@ -100,7 +100,7 @@ private JID getSelectedJID() {
 
         return user.getJID();
       } else {
-        return contacts.get(0);
+        return contacts.get(0).getBareJID();
```

Comment:
Why is it important that this uses the bare JID?

Reply:
I already explained this during the review that we just use bare JID for the chat.

1. If you want to send a message to an offline contact you must use the bare JID
2. If you use a bare JID the server is responsible to route this to the correct presence.

Reply:
My bad, didn't notice the class name.